M3T2/Tests/CategoriesTests.cs

```diff
@@ -0,0 +1,137 @@
+﻿public class CategoriesTests
```

Comment:
I decided to go with integration tests because fundamentally there is just nothing here to unit test in the sense that all the code is currently doing is interfacing with the database, so the only way to check whether it's correct is through integration testing, in my opinion. Pros: if the integration test passes, you know it works. Cons: you need to set up the environment to run tests and they run slower compared to unit tests.